agent/src/beerocks/slave/beerocks_slave_main.cpp

```diff
@@ -145,7 +145,8 @@ static std::string get_sta_iface_from_hostap_iface(const std::string &hostap_ifa
 }
 
 static void fill_son_slave_config(beerocks::config_file::sConfigSlave &beerocks_slave_conf,
-                                  son::slave_thread::sSlaveConfig &son_slave_conf, int slave_num)
+                                  son::slave_thread::sSlaveConfig &son_slave_conf,
```

Comment:
Commit message headline should be "agent: get hostap_iface from platform DB"

Commit message suggestion:

>agent: get hostap_iface from platform DB
>
> Preparative commit.
>This commit utilizes the new API to read all the hostap interfaces
from the platform database instead of the agent config file, so that each platform can have
its own different interface names.
Interfaces are read into an unordered map of radio index to interface name which is propagated to all of the agent threads but is still not used.
>
> Signed-off-by ...